CIP10/README.md

```diff
@@ -52,6 +52,7 @@ transaction_metadatum_label | description
 0 - 15                      | reserved*
 1967                        | nut.link metadata oracles registry
 1968                        | nut.link metadata oracles data points
+6770                        | fortunes.coconutpool.com forune teller
```

Comment:
assuming you mean "fortune teller" right? 
Again, thanks for rebasing the PR, or closing this one an opening a new one) as the structure of the directory has changed since this one was opened: '`/CIP-0010/CIP-0010.md`'

Reply:
Fortune Teller -- yes.